awx/ui_next/src/screens/Inventory/InventorySourceDetail/InventorySourceDetail.jsx

```diff
@@ -0,0 +1,255 @@
+    return () => {
+      isMounted.current = false;
+    };
+  }, []);
+
+  const handleDelete = async () => {
+    try {
+      await Promise.all([
+        InventorySourcesAPI.destroyHosts(id),
+        InventorySourcesAPI.destroyGroups(id),
+        InventorySourcesAPI.destroy(id),
+      ]);
+      history.push(`/inventories/inventory/${inventory.id}/sources`);
+    } catch (error) {
+      if (isMounted.current) {
+        setDeletionError(error);
+      }
+    }
+  };
+
+  const VERBOSITY = {
+    0: i18n._(t`0 (Warning)`),
+    1: i18n._(t`1 (Info)`),
+    2: i18n._(t`2 (Debug)`),
+  };
+
+  let optionsList = '';
+  if (
+    overwrite ||
+    overwrite_vars ||
+    update_on_launch ||
+    update_on_project_update
+  ) {
+    optionsList = (
+      <List>
+        {overwrite && <ListItem>{i18n._(t`Overwrite`)}</ListItem>}
+        {overwrite_vars && (
+          <ListItem>{i18n._(t`Overwrite variables`)}</ListItem>
+        )}
+        {update_on_launch && <ListItem>{i18n._(t`Update on launch`)}</ListItem>}
+        {update_on_project_update && (
+          <ListItem>{i18n._(t`Update on project update`)}</ListItem>
+        )}
+      </List>
+    );
+  }
+
+  return (
+    <CardBody>
+      <DetailList>
+        <Detail label={i18n._(t`Name`)} value={name} />
+        <Detail label={i18n._(t`Description`)} value={description} />
+        <Detail label={i18n._(t`Source`)} value={source} />
+        {organization && (
+          <Detail
+            label={i18n._(t`Organization`)}
+            value={
+              <Link to={`/organizations/${organization.id}/details`}>
+                {organization.name}
+              </Link>
+            }
+          />
+        )}
+        <Detail
+          label={i18n._(t`Ansible environment`)}
+          value={custom_virtualenv}
+        />
+        {source_project && (
+          <Detail
+            label={i18n._(t`Project`)}
+            value={
+              <Link to={`/projects/${source_project.id}/details`}>
+                {source_project.name}
+              </Link>
+            }
+          />
+        )}
+        <Detail label={i18n._(t`Inventory file`)} value={source_path} />
+        <Detail
+          label={i18n._(t`Custom inventory script`)}
+          value={source_script?.name}
+        />
+        <Detail label={i18n._(t`Verbosity`)} value={VERBOSITY[verbosity]} />
+        <Detail
+          label={i18n._(t`Cache timeout`)}
+          value={`${update_cache_timeout} ${i18n._(t`seconds`)}`}
+        />
+        {credentials?.length > 0 && (
+          <Detail
+            fullWidth
+            label={i18n._(t`Credential`)}
+            value={credentials.map(cred => (
+              <CredentialChip key={cred?.id} credential={cred} isReadOnly />
+            ))}
+          />
+        )}
+        {source_regions && (
+          <Detail
+            fullWidth
+            label={i18n._(t`Regions`)}
+            value={
+              <ChipGroup numChips={5}>
```

Comment:
Perhaps, we could rebase this code, and use the new custom ChipGroup.

See: https://github.com/ansible/awx/pull/6935